VERSION

```diff
@@ -1 +1 @@
-19.03.3-beta1
+19.03.3-beta2
```

Comment:
Either the changelog or this line is wrong

Reply:
fixed